vnet_manager/actions/manager.py

```diff
@@ -144,9 +144,10 @@ def preform_create_action(self):
         create_machines(self.config, machines=self._machines)
         # Put user requested file on the machines
         put_files_on_machine(self.config)
-        # Put /etc/hosts on the machines
-        generate_vnet_hosts_file(self.config)
-        place_vnet_hosts_file_on_machines(self.config)
+        if no_host is False:
```

Comment:
self.no_hosts

Reply:
Btw, this should just be `if self.no_hosts`. Not `if self.no_hosts if False`, that is a double negative, making it True by default.

Reply:
if `-no-hosts` is set, it sets that variable as a boolean "True". When the `-no-hosts` is not set, it is set as "False". So when that variable is "False" it should execute the `generate_vnet_hosts_file(..)` and `place_vnet_hosts_file_on_machines(..)`

Reply:
Yes, your right. Then only the `is False` remains. Because argeparse passes None as default. Might be nicer to note it `if not self.no_hosts`